src/js/common/schemaform/FormApp.jsx

```diff
@@ -62,14 +64,17 @@ export default class FormApp extends React.Component {
 
 
     return (
-      <div className="row">
-        <Element name="topScrollElement"/>
-        <div className="usa-width-two-thirds medium-8 columns">
-          {formConfig.title && !isIntroductionPage && <FormTitle title={formConfig.title} subTitle={formConfig.subTitle}/>}
-          {content}
-        </div>
-        <div className="usa-width-one-third medium-4 columns show-for-medium-up">
+      <div>
+        <div className="row">
+          <Element name="topScrollElement"/>
+          <div className="usa-width-two-thirds medium-8 columns">
+            {formConfig.title && !isIntroductionPage && <FormTitle title={formConfig.title} subTitle={formConfig.subTitle}/>}
+            {content}
+          </div>
         </div>
+        <AskVAQuestions>
```

Comment:
Could we hide this on the confirmation page? Doesn't seem useful there (there are already phone numbers and they've completed the form).